server/container_create_linux.go

```diff
@@ -135,6 +136,14 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 
 	// eventually, we'd like to access all of these variables through the interface themselves, and do most
 	// of the translation between CRI config -> oci/storage container in the container package
+
+	// TODO: eventually, this should be in the container package, but it's going through a lot of churn
+	// and SpecAddAnnotations is already passed too many arguments
```

Comment:
nit of a nit, ignore unless you get other changes
```suggestion
	// and SpecAddAnnotations is already passing too many arguments
```

Reply:
updated